src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -539,6 +539,23 @@
 
 "ExposureNotificationSetting_Dismiss_Action" = "Abbrechen";
 
+"ExposureNotificationSetting_GeneralError_Title" = "Fehler";
+
+"ExposureNotificationSetting_GeneralError_LearnMore_Action" = "Mehr erfahren";
```

Comment:
General question, bc I'm not sure what the default behavior is: Are alert action titles usually non-capitalized?

Reply:
In German you do not capitalize verbs like "erfahren", even if they are in a title... In English, you can say "Learn More", but "Learn more" is acceptable, too imho.